### ISSUE TITLE
Theme Switcher in 3.9.1

### ISSUE THREAD

**Issue by JAF62** [jimgraygit/Onion#176]
I'm new to Onion OS - so forgive me if I am missing something obvious ........
I installed 3.9 & then 3.9.1 yesterday - all good except I don't see "Onion Theme Switcher" in apps and no themes option in settings.
I have plenty of themes in the themes folder - but no way of applying them.

Please advise what my mistake might be - or tell me where I can obtain the theme switcher app.

Thanks in advance.

**Comment by dwmccoy** [comment]
If you do not see the Theme Switcher APP in the APP menu, Launch the Onion OS Installer APP and install the Theme Switcher APP from the APP section.

**Comment by JAF62** [comment]
I already tried that, but it doesn't appear in the list

**Comment by dwmccoy** [comment]
Sounds like your installation got broken.  If your trying to avoid a re-install, you could unzip the OnionOS 3.91 zip file and copy the .tmp_update\.data\Layer2\Theme switcher\App\ThemeSwitcher folder over to your /APP folder on your SDCARD to see if it resolves your issue.

**Comment by JAF62** [comment]
Thanks for the tip.

I did as you said and there was no difference.

So I launched the .sh file to launch it in dingux commander. That opens Theme Switcher and can now change themes.
But Theme Switcher still doesn't appear as an option in the installer - so the manual launch is my only option for now.

My assumption was also a faulty install, but have downloaded and executed 2 different instances of 3.9 and 3.9.1 on 2 different sd cards - so I can't see that the installation got broken - 35 years experience of IT  including Unix / Linux and a good few years of messing around with retro consoles - So at the moment I am baffled 😕

Thanks for helping me thus far 👍
